apps/examples/smartfs/fs_performance/fs_performance_main.c

```diff
@@ -0,0 +1,521 @@
+		printf("\n****************************************************************************************\n");
+		memset(g_read_buf, '1', MAX_RD_BUF_SIZE);
+
+		for (int file_size = 512; file_size <= g_fsz; file_size *= 2) {
+			if (g_bsz > file_size)
+				continue;
+
+			fd = open(g_file[0], O_WROK | O_CREAT);
+			if (fd < 0) {
+				printf("Unable to open file: %s, errno: %d\n", g_file[0], errno);
+				ret = ERROR;
+				goto close_with_frt;
+			}
+
+			ret = write(fd, g_rw_buf, file_size);
+			if (ret != (file_size)) {
+				printf("Unable to write in file: %s, ret: %d\n", g_file[0], ret);
+				ret = ERROR;
+				goto close_with_fd;
+			}
+			close(fd);
+
+			ret = ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&before);
+			if (ret < 0) {
+				goto error_with_frt_status;
+			}
+
+			for (i = 1; i <= ITR_READ; i++) {
+				fd = open(g_file[0], O_RDOK);
+				if (fd < 0) {
+					printf("Unable to open file: %s to read, fd: %d\n", g_file[0], fd);
+					ret = ERROR;
+					goto close_with_frt;
+				}
+
+				do {
+					ret = read(fd, g_read_buf, g_bsz);
+				} while (ret == g_bsz);
+
+				close(fd);
+			}
+
+			ret = ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&after);
+			if (ret < 0) {
+				goto error_with_frt_status;
+			}
+
+			tot_time += (after.timeleft - before.timeleft);
+			printf("End File size: %d bytes, Buffer size: %d bytes, Total time: %lu us\n", file_size, g_bsz, tot_time);
+
+			ret = unlink(g_file[0]);
+			if (ret < 0) {
+				printf("Failed to unlink file %s, errno: %d\n", g_file[0], errno);
+				ret = ERROR;
+				goto close_with_frt;
+			}
+		}
+		break;
+
+	case TEST_FLAG_GC_EFFECT :
+		printf("\n****************************************************************************************\n");
+		printf("**********************************TEST GC EFFECT************************************");
+		printf("\n****************************************************************************************\n");
+		for (i = 1; i <= ITR_GC; i++) {
+			ret = ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&before);
+			if (ret < 0) {
+				goto error_with_frt_status;
+			}
+
+			fd = open(g_file[0], O_WROK | O_CREAT);
+			if (fd < 0) {
+				printf("Unable to open file: %s, fd: %d\n", g_file[0], fd);
+				ret = ERROR;
+				goto close_with_frt;
+			}
+
+			ret = write(fd, g_rw_buf, g_fsz);
+			if (ret != (g_fsz)) {
+				 printf("Unable to write to file: %s, ret: %d\n", g_file[0], ret);
+				 ret = ERROR;
+				 goto close_with_fd;
+			}
+			close(fd);
+
+			ret = unlink(g_file[0]);
+			if (ret != OK) {
+				printf("Failed to unlink file: %s, errno: %d\n", g_file[0], errno);
+				ret = ERROR;
+				goto close_with_frt;
+			}
+
+			ret = ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&after);
+			if (ret < 0) {
+				goto error_with_frt_status;
+			}
+
+			itr_time = after.timeleft - before.timeleft;
+			printf("iteration %d, time: %lu us\n", i, itr_time);
```

Comment:
Let's print the unit of the time for the same results. line 216, line 279.

Reply:
updated